.github/workflows/ci_docker.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - develop
       - master
+      - gpl769_use_cherrypicked_view
```

Comment:
We need to remember to remove this after the merges.

Reply:
Ok, removed